src/content/en/updates/2016/10/devtools-digest.md

```diff
@@ -0,0 +1,79 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New Console features, updates on the context selector bug, and the new UC Browser user agent.
+
+{# wf_updated_on: 2016-10-18 #}
+{# wf_published_on: 2016-10-18 #}
+{# wf_tags: devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
```

Comment:
I **strongly** recommend the `wf_featured_snippet` tag, it's what's used for the description on the index page, vs the `description:` which is used as the `<meta description`. You can also include HTML in the `wf_featured_snippet` to be a little more attention grabbing.